docs/troubleshooting-guide/filters.md

```diff
@@ -0,0 +1,93 @@
+
+## My linked filter seems to have no effect
+
+**How to detect this:** After creating and applying a linked filter, you see the same set of rows that you saw before the second (linked) filter was applied.
+
+**How to fix this:** This is the inverse of the previous problem: all of the rows that pass the first test also pass the second. For example, if you create a date filter and then link a category filter to it, the second filter might have no effect. Again, you can check this by writing a native SQL query that you think should produce the reduced result; if it does not, the problem is most likely with the logic.
+
+FIXME: https://github.com/metabase/metabase/issues/14595
+
+## My linked filter widget does not display a dropdown of filtered values
+
+**How to detect this:** After linking Filter A to Filter B, you expect Filter B to display a dropdown showing only the values constrained by the current setting of Filter A. Instead, the dropdown shows all available values.
+
+**How to fix this:** In order for a linked filter widget to display the correct subset of values, an explicit [foreign key][foreign-key-glossary] definition must be set up---linking the filters does not by itself tell Metabase about the relationship. To check this, look at Metabase's data model for your database.
+
+FIXME: have I understood https://github.com/metabase/metabase/issues/16872 correctly?
+
+## My filter does not work when I embed my question
+
+**How to detect this:** A SQL question with a filter works correctly in Metabase itself, but the filter has no effect when the question is embedded via public sharing. For example, the question is `select {% raw %}{{filter}}{% endraw %}`, and filling in a constant value such as `1` works when the question is in Metabase, but is removed when the question is embedded in a web page using an iframe.
+
+**How to fix this:** Signed embedding must be used to pass filter settings to embedded questions.
+
+Depending on the programming language, you must ensure that an empty set of parameters is passed as an object so that they will be deserialized correctly. For example, in R the parameters must be:
+
+```
+params <- list()
+names(params) <- character(0)
+```
+
+while in PHP, they must be:
+
+```
+(object) []
+```
+
+Without this, the parameters are deserialized as an empty array rather than an empty object.
+
+FIXME:
+
+- https://github.com/metabase/metabase/issues/17019 talks about this - I'm pretty sure my discussion is wrong.
+- See also https://metabase.zendesk.com/agent/tickets/5798
+
+## FIXME
+
+Is there anything in these we can/should use?
+
+- https://metabase.zendesk.com/agent/tickets/5808 is more a how-to than a troubleshooting
+
+- https://metabase.zendesk.com/agent/tickets/3907 - need to do more with Field Filters
+
+- https://github.com/metabase/metabase/issues/15860 is a bug
```

Comment:
5808 doesn't have enough information, but short answer is "yes, it's possible".
3907 is described in the 2-3 sections above, but no response, so not sure, but probably issue 14595